internal/validator/beef/beef_validator.go

```diff
@@ -13,19 +13,19 @@ import (
 	"github.com/ordishs/go-bitcoin"
 )
 
-type BeefValidator struct {
+type Validator struct {
```

Comment:
The same as above. I can see why that makes sense, but question is, do we want it that way?

Reply:
IMO the redundant words are fine to remove
In cases of exceptions we can use `// nolint`